### PR TITLE
Fluff item for R3Valkyrie

### PR DESCRIPTION
**Pull request by FlattestGuitar**
```
/obj/item/clothing/head/valkyriehelmet
It's a cool-looking helmet.
For all jobs.
name: Rikki
ckey: R3Valkyrie
```

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -533,6 +533,14 @@
 	icon_state = "hairflowerp"
 	item_state = "hairflowerp"
```
Comment (Regen1) [inline code/modules/customitems/item_defines.dm]: hair flow erp

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -533,6 +533,14 @@
...
 
+/obj/item/clothing/head/valkyriehelmet //R3Valkyrie: Rikki
+	name = "charred visor"
+	desc = "A visor of alien origin, charred by fire and completely non-functioning. It's been impeccably polished, shiny!"
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "charred_visor"
+	species_restricted = list("Vox")
+	flags = BLOCKHAIR
```
Comment (r3valkyrie) [inline code/modules/customitems/item_defines.dm]: I'm having second thoughts about hiding hair. It actually looks pretty nice with hair.

Comment (KasparoVy) [inline code/modules/customitems/item_defines.dm]: This mask's sprite (and the sprite it's based off) was designed to show Vox quills.

**Comment by Necaladun** [comment]
And added to DB

**Comment by FlattestGuitar** [comment]
👌

**Pull request merged**